src/core/adapters/adaptlocationbias.js

```diff
@@ -0,0 +1,20 @@
+import LocationBias from '../models/locationbias';
+/*
+ * Constructs an SDK LocationBias model from an answers-core LocationBias
+ *
+ * @param {LocationBias} locationBias from answers-core
+ * @returns {@link LocationBias}
+ */
+export default function adaptLocationBias (locationBias) {
+  if (!locationBias) {
+    return new LocationBias({
+      accuracy: 'UNKNOWN'
+    });
+  }
+  return new LocationBias({
+    accuracy: locationBias.method,
+    latitude: locationBias.latitude,
+    longitude: locationBias.longitude,
+    locationDisplayName: locationBias.displayName
+  });
+}
```

Comment:
Would it make sense to use the existing LocationBias.from() here?

Reply:
They are a little different since the method and display name fields are different, but It might make sense to just replace the current 'from' methods with all the adapter code